classes/LineManagerNotifier.php

```diff
@@ -46,7 +50,13 @@ public function SendMessage($description, $user_ids)
 				$params['recipients'] = $all_except_last_recipient_csv . ' ' . lmsg('thankyou.grammar.list.and') . ' ' . $last_recipient;
 			}
 
-			NotificationMessage::Send('thankyou.new_thanks_manager', $params, [$line_manager], Constants::IM_TYPE_THANKYOU, null, $author_id);
+			try
+			{
+				NotificationMessage::Send('thankyou.new_thanks_manager', $params, [$line_manager], ThankYous::IM_TYPE_THANKYOU, null, $author_id);
+			} catch (Exception $exception)
+			{
+				throw new LogicException("Unexpected Exception thrown in NotificationMessage::Send", null, $exception);
+			}
```

Comment:
This too much of a catch-all and should simply be left to throw, or just caught and logged. I'd suggest logging instead of throwing.

Reply:
Fixed in cab767454b82fdcbeb059e124a0443e1639f49d6

Worth a review.

Reply:
Looks good!